mcs/build/tests.make

```diff
@@ -130,6 +126,10 @@ test-local-aot-compile: $(topdir)/build/deps/nunit-$(PROFILE).stamp
 
 endif # ALWAYS_AOT
 
+ifdef COVERAGE
+TEST_COVERAGE_FLAGS = -O=-aot --profile=coverage:output=$(topdir)/class/lib/$(PROFILE_DIRECTORY)/coverage_nunit_$(ASSEMBLY).xml
```

Comment:
should we output it next to the test assembly instead, i.e. next to `TestResult-$(PROFILE).xml` ?

Reply:
we need to collect them all (e.g. for all tests) later and feed them to reporter which would be a pain to do from individual folders

Reply:
not really, we do the same with the `TestResult-$(PROFILE).xml` + some globs on Jenkins already.

we need to gather the coverage xml from different platforms anyway and merge them into one report so we need a separate job anyway.

Reply:
I think it's better to grab them from a single folder (whatever that is) instead of searching for all XML files under mono/mcs/**. This also makes it more compatible with out-of-tree builds.